example/docker-compose/docker-compose.integrations.yaml

```diff
@@ -15,7 +15,7 @@ services:
   #
 
   grafana:
-    image: grafana/grafana:6.6.1
+    image: grafana/grafana:7.5.4
```

Comment:
This syncs with the default docker-compose file which was using a newer version of Grafana.